config/config.go

```diff
@@ -37,8 +37,9 @@ type Config struct {
 	}
 
 	Core struct {
-		Image string
-		Path  string
+		Image    string
```

Comment:
I don't get this part, why do we need name and rootpath? 
What rootpath means - consider remaining it to ConfigPath as this is what you mean right?
Also when the path is explicit given the app shouldn't create new directories (`.mesg`) under it.

Reply:
The goal of that is to have the database stored in it's own directory so the root path is the one that will store all the different "core" instances

Right now we have 
```
~/.mesg/
├── database
```

And this change to
```
~/.mesg/
├── mesg-core/
|   ├── database
```
Like that we can now run different instances of the core with their own separated database
```
~/.mesg/
├── mesg-core1/
|   ├── database
├── mesg-core2/
|   ├── database
...
```

The `RootPath` is the one to store the `.mesg` and the `CorePath()` is the one to use to know where to write. This one returns the `RootPath + Name`.

`ConfigPath` is a bit ambiguous because it's the config of the core so this should be the `~/.mesg/mesg-core`

> Also when the path is explicit given the app shouldn't create new directories (.mesg) under it.

Why shouldn't we do that ? The goal of this property is to be the "namespace" for the folder

Reply:
Based on the discussion on discord and the feedbacks of @krhubert we can remove the `RootPath` and just keep a `CorePath` that is customizable by the user and so we would just have the `MESG_CORE_PATH` to edit more